### PR TITLE
[#48379] Rename 'Working days' and add second level of navigation

### PR DESCRIPTION
**Pull request by mrmir**
https://community.openproject.org/work_packages/48379

### CODE REVIEW

File: app/controllers/admin/settings/date_format_settings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+module Admin::Settings
+  class DateFormatSettingsController < ::Admin::SettingsController
+    menu_item :date_format
+
+    before_action :validate_start_of_week_and_first_week_of_year_combination, only: :update
+
+    def update # rubocop:disable Lint/UselessMethodDefinition
+      super
+    end
```
Comment (oliverguenther) [inline app/controllers/admin/settings/date_format_settings_controller.rb:37]: This method can simply be removed?

**Review by oliverguenther — CHANGES_REQUESTED** [review]
The one mentioned method can be removed if I'm not mistaken. Good to merge afterwards.

File: app/controllers/admin/settings/date_format_settings_controller.rb (hunk 1 above)
Comment (mrmir) [reply app/controllers/admin/settings/date_format_settings_controller.rb:37]: I added that in mainly to suppress this rubocop error https://github.com/opf/openproject/runs/14132563725, but then it looks like I got the issue with Lint/UselessMethodDefinition anyway. Better to remove it and disable Rails/LexicallyScopedActionFilter instead?

Comment (oliverguenther) [reply app/controllers/admin/settings/date_format_settings_controller.rb:37]: Ah I see, okay. Hm I'm fine with either way. Feel free to merge as is then

**Review by oliverguenther — APPROVED** [review]

**Pull request merged**